src/gameStateService/game-state-service.ts

```diff
@@ -188,8 +188,7 @@ export class GameStateService extends PubSub {
     levelEndSceneData({levelEndData, data}) {
         this.levelEndData = {...levelEndData};
         this.data = data;
-        this.isLastLevel =
-        levelEndData.currentLevel === data.levels[data.levels.length - 1].levelMeta.levelNumber;
+        this.isLastLevel = levelEndData.currentLevel === data.levels[data.levels.length - 1].levelMeta.levelNumber;
```

Comment:
Please add test case for this one.

Reply:
@miguelccodev added test case for this one. please check. thank you